packages/Confirmation/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@paprika/confirmation",
+  "version": "0.1.0",
+  "description": "Confirmation component",
+  "author": "@paprika",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/acl-services/paprika/tree/master/packages/confirmation"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@babel/runtime-corejs2": "^7.3.1",
+    "@paprika/button": "^0.1.1",
+    "@paprika/heading": "^0.1.5",
+    "@paprika/helpers": "^0.1.0",
+    "@paprika/l10n": "^0.2.0",
+    "@paprika/popover": "^0.1.7",
+    "@paprika/stylers": "^0.1.1",
+    "@paprika/tokens": "^0.1.1",
+    "prop-types": "^15.7.2",
+    "uuid": "^3.3.2"
+  },
+  "peerDependencies": {
+    "react": "^16.8.4",
+    "react-dom": "^16.8.4",
+    "styled-components": "^4.2.0",
+    "@paprika/icon": "^0.2.0"
```

Comment:
Why make icons a peer dependency?

Reply:
Oh, i've moved to `devDependencies` as that is more appropriate, This was only need for a storybook example.

Reply:
For Storybook, you can just rely on the `package.json` in the root of the Paprika repo.  As long as its dependencies include this latest version of `icon`, you should be good.